package.json

```diff
@@ -31,6 +32,7 @@
     "react-dom": "^18",
     "react-google-recaptcha": "^3.1.0",
     "react-hook-form": "^7.53.0",
+    "react-refractor": "^3.1.1",
```

Comment:
i do not see where we use this also

Reply:
need to remove this one as well, xixixix